src/main/java/org/jenkinsci/plugins/github/webhook/WebhookManager.java

```diff
@@ -73,7 +76,7 @@ public static WebhookManager forHookUrl(URL endpoint) {
      * @return runnable to create hooks on run
      * @see #createHookSubscribedTo(List)
      */
-    public Runnable registerFor(final AbstractProject<?, ?> project) {
+    public Runnable registerFor(final Job<?, ?> project) {
```

Comment:
This change makes the Javadoc comment in line 133 obsolete, and without that, AbstractProject is an unused import as well.